feconf.py

```diff
@@ -578,6 +578,8 @@ def get_empty_ratings():
 LEARNER_DASHBOARD_URL = '/learner_dashboard'
 LEARNER_DASHBOARD_DATA_URL = '/learnerdashboardhandler/data'
 LEARNER_DASHBOARD_FEEDBACK_THREAD_DATA_URL = '/learnerdashboardthreadhandler'
+LEARNER_PLAYLIST_DATA_URL = '/learnerplaylistactivityhandler'
+LEARNER_INCOMPLETE_ACTIVITY_DATA_URL = '/incompleteactivityhandler'
```

Comment:
Seems odd to omit the "learner" prefix if we are using it above. /learnerincompleteactivityhandler perhaps?